### ISSUE TITLE
IDISP001 should not warn when using declaration

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#254]
```cs
        public static void Save(string fileName)
        {
            using var stream = new FileStream(fileName, FileMode.Create);
        }
```

**Comment by JohanLarsson** [comment]
No bug, was using 2.1.2 that does not know what using declaration is.
